bodhi/server/models.py

```diff
@@ -1483,6 +1487,20 @@ def unpush(self, koji):
                         self.nvr, tag, release.candidate_tag))
                 koji.moveBuild(tag, release.candidate_tag, self.nvr)
 
+    def is_latest(self) -> bool:
+        """Check if this is the latest build available in the stable tag."""
+        koji_session = buildsys.get_session()
+        builds = koji_session.getLatestBuilds(
+            self.update.release.stable_tag,
+            package=self.package.name
+        )
+
+        for build in builds:
+            build_creation_time = datetime.fromisoformat(build['creation_time'])
```

Comment:
This will be probably better :-)
```suggestion
            build_creation_time = build.get_creation_time()
```

Reply:
That would be neat but here build is a dictionary retruned by koji and not a Build type from Bodhi, the name are confusing here so I might rename it in koji_build.

Reply:
Yeah, it's confusing